tutorials/Tutorial 8 - Directed Hypergraphs.ipynb

```diff
@@ -0,0 +1,442 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    DH.add_edge([{1, 2, 3}, {3, 4}])  # Notice that the senders and reviewers need not be disjoint.
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done.</p>

Reply:
See comment below.